packages/oscal-react-library/src/components/OSCALAnchorLinkHeader.tsx

```diff
@@ -12,7 +12,16 @@ const AnchorLinkIcon = styled(LinkIcon)(({ theme }) => ({
   color: theme.palette.grey[500],
 }));
 
-export default function OSCALAnchorLinkHeader(props) {
+export interface AnchorLinkProps {
+  urlFragment?: string;
+}
+
+export interface OSCALAnchorLinkHeaderProps {
+  children: any;
```

Comment:
Do we just take in `children` because that's what React elements do? I'll wait for the docs but it feels like we can do something smarter here.

Reply:
I am still confused why this can't be `ReactNode`